### ISSUE TITLE
Improve the "403 Unauthorized" error page in the UI

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-registry#907]
Right now there is a generic error page for most server errors.  We should have a specialized error page for HTTP 403.

**Comment by carlesarnal** [comment]
A good improvement to this could be to disable some actions based on the configuration instead of just redirect or show an error page.

**Comment by EricWittmann** [comment]
Yep - there is a separate GH issue for that.  Which is why this issue is on the backlog and the other one is Accepted. :)
